modules/contracts/adapters/dex/src/exchanges/exchange_resolver.rs

```diff
@@ -56,10 +54,8 @@ pub(crate) fn resolve_exchange(value: &str) -> Result<Box<dyn DexCommand>, DexEr
         abstract_astroport_adapter::ASTROPORT => {
             Ok(Box::<abstract_astroport_adapter::dex::Astroport>::default())
         }
-        #[cfg(feature = "bow")]
-        abstract_kujira_adapter::KUJIRA => {
-            Ok(Box::<abstract_kujira_adapter::dex::Kujira>::default())
-        }
+        #[cfg(feature = "fin")]
+        abstract_kujira_adapter::KUJIRA => Ok(Box::<abstract_kujira_adapter::dex::Fin>::default()),
```

Comment:
We probably also want to change this match to match on "fin"?

Reply:
What do you mean?

Reply:
Yes instead of Kujira

Reply:
> Yes instead of `abstract_kujira_adapter::KUJIRA`

Sure, didn't expect we want to change that

Reply:
I think we want to, to stay consistent.